integrated_channels/sap_success_factors/exporters/learner_data.py

```diff
@@ -26,7 +26,14 @@ class SapSuccessFactorsLearnerExporter(LearnerExporter):
     Class to provide a SAPSF learner data transmission audit prepared for serialization.
     """
 
-    def get_learner_data_records(self, enterprise_enrollment, completed_date=None, grade=None, is_passing=False):
+    def get_learner_data_records(
+            self,
+            enterprise_enrollment,
+            completed_date=None,
+            grade=None,
+            is_passing=False,
+            grade_percent=None
```

Comment:
I'd be shocked if you don't have to upload a hell of a lot of tests with this additional field. Godspeed.

Reply:
... that's just what I spent all Friday afternoon doing 😒

Reply:
I'm from the future- tests have been updated! But I have some qualms about how the base integrated channel tests transmitting learner data